packages/mutator-io-plugin-out-dynamodb/README.md

```diff
@@ -12,6 +12,8 @@ Ideally this should leverage Rx.js to perform fail-safe operations like batchWri
 
 The configuration required is [exactly the same of the original sdk](https://github.com/aws/aws-sdk-js/blob/master/lib/dynamodb/document_client.d.ts).
 
+There's an extra custom parameter called `IGNORE_ERROR_CODES` which is a list of error codes that we might want to ignore to avoid bloating the logs (e.g. `ConditionalCheckFailedException` might not be a "real" exception so we can ignore it).
```

Comment:
Can not we make this more generic and include an optional field for messages?

For example, an item size limitation exceeded error can look something like this:
```
{
  code: 'ValidationException',
  message: 'Item size to update has exceeded the maximum allowed size'
}
```

But `ValidationException` can stand for other kind of exceptions as well. From the DynamoDB documentation:
```
ValidationException
Message: Varies, depending upon the specific error(s) encountered

This error can occur for several reasons, such as a required parameter that is missing, a value that is out range, or mismatched data types. The error message contains details about the specific part of the request that caused the error.
```